README.md

```diff
@@ -51,7 +51,7 @@ Once system dependencies have been satisfied, you can install the library with t
 pip install spokestack
 ```
 
-## Development
+## Using the Library
```

Comment:
Yeah, that works. I'd personally go with "usage" (shorter/consistent with other platforms), but this means the same.